hubble/src/indexer/eth/context.rs

```diff
@@ -0,0 +1,23 @@
+use std::fmt::Display;
+
+use url::Url;
+
+#[derive(Clone)]
+pub struct EthContext {
+    pub urls: Vec<Url>,
+}
+
+impl Display for EthContext {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "urls: {}",
+            self.urls
+                .iter()
+                .enumerate()
+                .map(|(index, url)| format!("{}: {}", index, url.as_str().to_string()))
+                .collect::<Vec<_>>()
+                .join(", ")
+        )
+    }
+}
```

Comment:
nit: what are these display impls used for? seems like this should actually be debug?

Reply:
formatting log messages. yes, could also be debug.

can I assume: if only used in logging, then only implement Debug?